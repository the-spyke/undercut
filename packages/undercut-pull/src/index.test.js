import { describe, expect, test } from "@jest/globals";

import * as exports from "./index.js";

describe(`Exports`, () => {
	test(`of the main "pull" entry should be stable`, () => {
		expect(Object.keys(exports).sort()).toMatchSnapshot();
	});
});
