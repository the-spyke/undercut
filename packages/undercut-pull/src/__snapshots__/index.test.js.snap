// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Exports of the main "pull" entry should be stable 1`] = `
Array [
  "append",
  "average",
  "buffer",
  "bufferAll",
  "chunk",
  "collect",
  "collectArray",
  "collectMap",
  "collectObject",
  "collectSet",
  "compact",
  "composeOperations",
  "concatEnd",
  "concatStart",
  "count",
  "difference",
  "differenceBy",
  "every",
  "filter",
  "find",
  "findIndex",
  "first",
  "flatMap",
  "flatten",
  "flattenArrays",
  "flattenIterables",
  "forEach",
  "groupBy",
  "includes",
  "interleave",
  "intersection",
  "intersectionBy",
  "join",
  "last",
  "map",
  "max",
  "min",
  "nth",
  "orderBy",
  "prepend",
  "pull",
  "pullArray",
  "pullLine",
  "pullValue",
  "range",
  "reduce",
  "remove",
  "reverse",
  "skip",
  "skipWhile",
  "some",
  "sort",
  "sortNumbers",
  "sortStrings",
  "sum",
  "symmetricDifference",
  "symmetricDifferenceBy",
  "take",
  "takeWhile",
  "toArray",
  "toConsumer",
  "toMap",
  "toNull",
  "toObject",
  "toObserver",
  "toSet",
  "toValue",
  "union",
  "unionBy",
  "unique",
  "uniqueBy",
  "unzip",
  "unzipWith",
  "zip",
  "zipWith",
]
`;
